frontend/integration-tests/tests/devconsole/git-import-flow.scenario.ts

```diff
@@ -0,0 +1,56 @@
+import { browser, element, by, ExpectedConditions as until } from 'protractor';
+import { appHost, checkLogs, checkErrors } from '../../protractor.conf';
+import {
+  navigateImportFromGit,
+  setBuilderImage,
+  enterGitRepoUrl,
+  addApplication,
+  applicationName,
+  appName,
+  builderImage,
+  buildImageVersion,
+  createButton,
+} from '../../views/devconsole-view/git-imort-flow';
+import { newApplicationName, newAppName } from '../../views/devconsole-view/new-app-name.view';
+import { switchPerspective, Perspective } from '../../views/devconsole-view/dev-perspective.view';
+import { sideHeader } from '../../tests/devconsole/dev-perspective.scenario';
+
+export const version = element(by.id('8-RHOAR-link'));
```

Comment:
Can you move this to ```devconsole-view/git-imort-flow.ts``` file, scenario file should not have any export. and ```version``` is too generic name can you change it to ```builderImageVersion```